Rakefile

```diff
@@ -235,7 +209,7 @@ end
 namespace :lint do
   desc 'Automatically corrects style errors where possible'
   task autocorrect: %w[dependencies:lint:check] do
-    swiftlint %w[autocorrect]
+    swiftlint %w[lint --autocorrect --quiet]
   end
 end
```

Comment:
I think this could become a first level task. `rake format`. But I'll leave it for another PR, maybe.